src/libraries/System.Private.CoreLib/src/System/SR.cs

```diff
@@ -21,7 +21,7 @@ internal static partial class SR
         // Needed for debugger integration
         internal static string GetResourceString(string resourceKey)
         {
-            return GetResourceString(resourceKey, string.Empty);
+            return GetResourceString(resourceKey, null);
```

Comment:
@tarekgh I made this change due to the discussion https://github.com/dotnet/runtime/issues/37587#issuecomment-640942150

Do you mind taking a look at this?

Reply:
Is it possible to get rid of this file and use the one in the common? we can look quickly at the differences. both should be providing the same logic. or if you want we can do that in a separate PR.

Reply:
This file is extension of the common file, we cannot get rid of it

Reply:
From the discussion on that issue it may be that it there is a problem as some codegen from arcade might be calling this method specifically so removing might not be so simple, although I may have misinterpreted the discussion. I would prefer to scope the change to only this for now so that if Arcade gets broken by removing the method we don't have to revert this whole PR

Reply:
> This file is extension of the common file, we cannot get rid of it

I may have misread but I thought what Tarek meant was more to remove this method, in favor of the one in the other overload defined in Common.

Reply:
if this is the case, the SR.cs that in the common already have the method: 

  internal static string GetResourceString(string resourceKey, string? defaultString = null)

so I guess we should delete GetResourceString from here then. 

---
In reply to: [445786809](https://github.com/dotnet/runtime/pull/38397#discussion_r445786809) [](ancestors = 445786809)

Reply:
Also the SR.cs in common is just ifdef stuff for corelib which we can get rid of now as the ResourceManager became a part of corelib already. that means we can get rid of SR.cs which is in the corelib. as I mentioned we can make this in other PR if we need to.

---
In reply to: [445792039](https://github.com/dotnet/runtime/pull/38397#discussion_r445792039) [](ancestors = 445792039,445786809)

Reply:
I don't think such change should be part of this PR